chrome-launcher/.npmignore

```diff
@@ -0,0 +1,20 @@
+
+# folders
+.vscode/
+test/
+node_modules/
+
+# include all compiled JS
+!*.js
```

Comment:
[the ignore files won't be merged](https://github.com/npm/npm/wiki/Files-and-Ignores#details-1) though, is the parent folder ignore creeping in?